api/src/certification/configuration/application/sco-whitelist-route.js

```diff
@@ -40,6 +40,28 @@ const register = async function (server) {
         ],
       },
     },
+    {
+      method: 'GET',
+      path: '/api/admin/sco-whitelist',
+      config: {
+        pre: [
+          {
+            method: (request, h) =>
+              securityPreHandlers.hasAtLeastOneAccessOf([securityPreHandlers.checkAdminMemberHasRoleSuperAdmin])(
```

Comment:
Lui seul suffit je pense 
```suggestion
              securityPreHandlers.checkAdminMemberHasRoleSuperAdmin
```

Reply:
Done